RealmSwift/Combine.swift

```diff
@@ -65,10 +65,13 @@ public protocol RealmSubscribable {
     /// :nodoc:
     func _observe<S>(on queue: DispatchQueue?, _ subscriber: S)
         -> NotificationToken where S: Subscriber, S.Input == Self, S.Failure == Error
-    // swiftlint:enable identifier_name
+    /// :nodoc:
+    func _observe<S>(_ subscriber: S, _ keyPaths: [String]?)
+        -> NotificationToken where S: Subscriber, S.Input == Void, S.Failure == Never
     /// :nodoc:
     func _observe<S>(_ subscriber: S)
         -> NotificationToken where S: Subscriber, S.Input == Void, S.Failure == Never
+    // swiftlint:enable identifier_name
```

Comment:
`_observe()` should just be added to the allowed names in .swiftlint.yml rather than excluding it on each use.